repositories.bzl

```diff
@@ -37,115 +82,78 @@ IO_GRPC_GRPC_JAVA_OVERRIDE_TARGETS = {
     "io.grpc:grpc-testing": "@io_grpc_grpc_java//testing",
 }
 
-def grpc_java_repositories(
-        omit_com_google_android_annotations = False,
-        omit_com_google_api_grpc_google_common_protos = False,
-        omit_com_google_auth_google_auth_library_credentials = False,
-        omit_com_google_auth_google_auth_library_oauth2_http = False,
-        omit_com_google_code_findbugs_jsr305 = False,
-        omit_com_google_code_gson = False,
-        omit_com_google_errorprone_error_prone_annotations = False,
-        omit_com_google_guava = False,
-        omit_com_google_guava_failureaccess = False,
-        omit_com_google_j2objc_j2objc_annotations = False,
-        omit_com_google_protobuf = False,
-        omit_com_google_protobuf_java = False,
-        omit_com_google_protobuf_javalite = False,
-        omit_com_google_truth_truth = False,
-        omit_com_squareup_okhttp = False,
-        omit_com_squareup_okio = False,
-        omit_io_grpc_grpc_proto = False,
-        omit_io_netty_buffer = False,
-        omit_io_netty_common = False,
-        omit_io_netty_transport = False,
-        omit_io_netty_transport_native_epoll = False,
-        omit_io_netty_codec = False,
-        omit_io_netty_codec_socks = False,
-        omit_io_netty_codec_http = False,
-        omit_io_netty_codec_http2 = False,
-        omit_io_netty_handler = False,
-        omit_io_netty_handler_proxy = False,
-        omit_io_netty_resolver = False,
-        omit_io_netty_tcnative_boringssl_static = False,
-        omit_io_opencensus_api = False,
-        omit_io_opencensus_grpc_metrics = False,
-        omit_io_perfmark = False,
-        omit_javax_annotation = False,
-        omit_junit_junit = False,
-        omit_org_apache_commons_lang3 = False,
-        omit_org_codehaus_mojo_animal_sniffer_annotations = False):
+def grpc_java_repositories():
     """Imports dependencies for grpc-java."""
-    if not omit_com_google_android_annotations:
+    if not native.existing_rule("com_google_protobuf"):
+        com_google_protobuf()
+    if not native.existing_rule("com_google_protobuf_javalite"):
+        com_google_protobuf_javalite()
+    if not native.existing_rule("io_grpc_grpc_proto"):
```

Comment:
Not in dictionary order?

Reply:
I don't understand. I don't see anything wrong here and below.

Reply:
I mean the artifact names `"com_google_protobuf_javalite"`, `"io_grpc_grpc_proto"`, `"com_google_android_annotations"`... are not sorted in alphabetic order. It will be easier to look up if listing them in alphabetic order.

Reply:
Oh, you are comparing with annotations. These three were pulled from the rest because they are different (and see the extra newline after io_grpc_grpc_proto). These three are Bazel-aware repositories, the rest are Maven repositories. Once we require maven_install we will delete the rest, but leave these.